internal/platform/time_test.go

```diff
@@ -44,25 +43,10 @@ func Test_Walltime(t *testing.T) {
 }
 
 func Test_Nanotime(t *testing.T) {
-	tests := []struct {
```

Comment:
maybe make this two different tests. the cheaper one with old logic for Test_nanotimePortable, then the new integration test that sleeps for some period larger than nanos to accommodate pauses, etc. ideally not 1 second.